src/js/components/Avatar/stories/typescript/Stacked.tsx

```diff
@@ -43,14 +42,14 @@ export const Stacked = () => {
       <Avatar background="accent-2" margin={{ left: 'medium' }}>
```

Comment:
There are still some colors in this story that aren't available in the hpe theme (any of the "accent-*" colors)